src/test/java/net/snowflake/client/jdbc/CustomProxyLatestIT.java

```diff
@@ -496,21 +532,23 @@ public PasswordAuthentication getPasswordAuthentication() {
     // Enable these parameters to use JVM proxy parameters instead of connection string proxy
     // parameters.
     // Connection parameters override JVM  proxy params, so these incorrect params won't cause
-    // failures IF
-    // connection proxy params are enabled and working.
-    System.setProperty("http.useProxy", "true");
-    System.setProperty("http.proxyHost", "fakehost");
-    System.setProperty("http.proxyPort", "8081");
-    System.setProperty("https.proxyHost", "fakehost");
-    System.setProperty("https.proxyPort", "8081");
+    // failures IF connection proxy params are enabled and working.
+    if (usesIncorrectJVMProperties) {
+      System.setProperty("http.useProxy", "true");
+      System.setProperty("http.proxyHost", "fakehost");
+      System.setProperty("http.proxyPort", "8081");
+      System.setProperty("https.proxyHost", "fakehost");
+      System.setProperty("https.proxyPort", "8081");
+    }
 
     // SET USER AND PASSWORD FIRST
-    String user = "USER";
-    String passwd = "PASSWORD";
+    String user = "mknister";
+    String passwd = "Argumentc1inicspam!";
```

Comment:
You'd better to change these two lines back because some password detection tool make think it is a secret leak.